web/sites/default/config/core.entity_form_display.node.park_facility.default.yml

```diff
@@ -224,6 +224,7 @@ third_party_settings:
     group_amenities:
       children:
         - field_park_amenities_activities
+        - field_pets_amenities
```

Comment:
This field appears to be a leftover from an earlier commit. Could you check if the form display needs to be updated for this story?